ui/app/components/database-role-edit.js

```diff
@@ -67,6 +67,19 @@ export default class DatabaseRoleEdit extends Component {
     }
     return warnings;
   }
+
+  get skipImport() {
+    const backend = this.args.model?.backend;
+    const dbs = this.args.model?.database || [];
+    if (!backend || dbs.length === 0) {
+      return null;
+    }
+    return this.store
+      .queryRecord('database/connection', { id: dbs[0], backend })
+      .then((record) => record.skip_static_role_rotation_import)
+      .catch(() => null);
```

Comment:
This adds another network request to the same endpoint, right? Since this logic is essentially the same as `get databaseType` I wonder if we can consolidate it so that we're only making a request 🤔 

One option is to update `databaseType` to return an object of pertinent params

```js
  get databaseParams() {
    const backend = this.args.model?.backend;
    const dbs = this.args.model?.database || [];
    if (!backend || dbs.length === 0) {
      return null;
    }
    return this.store
      .queryRecord('database/connection', { id: dbs[0], backend })
      .then(({ plugin_name, skip_static_role_rotation_import }) => { plugin_name, skip_static_role_rotation_import })
      .catch(() => null);
  }
  ```
and then update the params passed to `DatabaseRoleSettingForm`